lib/get-profile.js

```diff
@@ -53,11 +54,14 @@ module.exports.getProfile = async zip => {
     withOutputPath(profile, './src/images/icon.png'),
     avatarBuffer
   )
-  await fs.writeFile(
-    withOutputPath(profile, './src/images/avatar.png'),
-    avatarBuffer
-  )
 
+  Jimp.read(avatarBuffer, (err, avatar) => {
```

Comment:
Can we change that to a promise and `await` it?

Reply:
```
 let avatar = await Jimp.read(avatarBuffer)
      avatar
        .resize(150, 150) // resize
        .write(withOutputPath(profile, './src/images/avatar.png')); // save
```

Reply:
I think you also need to await the `write`:
```suggestion
  const jimp = await Jimp.read(avatarBuffer)
  await jimp.resize(150, 150).write(withOutputPath(profile, './src/images/avatar.png'))
```